package.json

```diff
@@ -47,6 +47,7 @@
   "dependencies": {
     "bn": "^1.0.1",
     "ethereumjs-util": "^5.1.1",
+    "flexboxgrid": "^6.3.1",
```

Comment:
Ahh yes that was third thing I wanted to show you in my other PR. We can use react version if there problems with it we can always go back to css.  
Here is [example](https://github.com/Neufund/generic-ico/blob/basic_layouts/src/components/HeaderInitial.js#L11)